off-chain/src/Network/Leader.purs

```diff
@@ -427,12 +438,12 @@ resetLeaderState leaderNode = do
   setToRefAtLeaderState leaderNode OrderedMap.empty _.waitingForSignature
   setToRefAtLeaderState leaderNode OrderedMap.empty _.waitingForSubmission
   setToRefAtLeaderState leaderNode WaitingForActions _.stage
```

Comment:
Do we need this stage flag at all?

Reply:
it's user when we report the statu of the server, the original idea was that we can use it together with `actionStatus` to make us an idea of how long it would take for the server to dispatch (but we aren't providing a way rigth now)